subsys/partition_manager/pm.yml.memfault

```diff
@@ -3,4 +3,9 @@
 memfault_storage:
   placement: {before: [tfm_storage, end]}
   size: CONFIG_PM_PARTITION_SIZE_MEMFAULT_STORAGE
+#ifdef CONFIG_BUILD_WITH_TFM
+  align: {start: CONFIG_NRF_SPU_FLASH_REGION_SIZE}
+#else
   align: {start: CONFIG_MEMFAULT_NCS_FLASH_REGION_SIZE}
```

Comment:
Not the expert here. But it seems the value set via CONFIG_MEMFAULT_NCS_FLASH_REGION_SIZE is ignored when building with TFM. Not sure this is acceptable and what the side effects are. Can the build system be made to throw a warning when this happens?

Reply:
The MEMFAULT align value is on the erase-block-size of the flash. I think it is a safe assumption that the SPU region size is aligned, but a super-set of the erase-block-size.